docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -394,6 +394,151 @@ The default is `true`.
     You can override the default DNS server if it's unreliable or doesn't serve TCP DNS requests, using the
     [experimental option](#xhelp) `--Xp2p-dns-discovery-server=<HOST>`.
 
+### `engine-rpc-http-port`
+
+=== "Syntax"
+
+    ```bash
+    --engine-rpc-http-port=<PORT>
+    ```
+
+=== "Example"
+
+    ```bash
+    --engine-rpc-http-port=8550
+    ```
+
+=== "Environment variable"
+
+    ```bash
+    BESU_ENGINE_RPC_HTTP_PORT=8550
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    engine-rpc-http-port=8550
+    ```
+
+The listening port for the Engine API calls (`ENGINE`, `ETH`) for JSON-RPC over HTTP.
```

Comment:
@jflo, Are Engine API calls, calls made from a consensus client (Teku)? Are these a seperate set of APIs that are implemented for the merge?